docs/reference/contributing.rst

```diff
@@ -0,0 +1,17 @@
+================================
+How to Contribute to the Project
+================================
+
```

Comment:
My only comment is to make this the same as the "CONTRIBUTING.md" file above. The CONTRIBUTING.md file has added text in it.

Reply:
I updated the contributing in docs to match with the `CONTRIBUTING.md`